stories/src/primatives/quote-list.jsx

```diff
@@ -18,7 +18,7 @@ const Wrapper = styled.div`
   display: flex;
   flex-direction: column;
   opacity: ${({ isDropDisabled }) => (isDropDisabled ? 0.5 : 'inherit')};
-  padding: ${grid}px;
```

Comment:
I am not sure if it is feasible to detect for margin collapsing for droppable placeholders. My current thinking is to add a note about it in the docs and leave it at that. Thoughts?

Reply:
might have to be. I'm spiking a solution atm but I'm not sure if it'll cover everything. I'll report back 👨‍🚀